blog/content/post/2020-12-09-OSFC_2020.md

```diff
@@ -0,0 +1,198 @@
+beginning big kudos to 9elements for organizing the event and gather crucial
+voices of contributors from various firmware community corners. We want to share
+some thoughts that evolved during the talks and send kudos to many people who
+made this conference happen. We will leave a few notes on every attended talk,
+from day 1 to day 3, according to the
+[schedule](https://cfp.osfc.io/osfc2020/schedule/), trying to gather talks with
+references and sources. You can go through the post or skip to the chosen
+conference. Remember, presented descriptions are only our interpretation, not
+official statements of presenters. To watch the full video and get your own
+opinion, click the chosen title.  
+
+### Day 1
+
+* [Why LVFS is Essential for OEMs](https://vimeo.com/488137809) by [Richard Hughes](https://twitter.com/hughsient)
+
+The Linux Vendor Firmware Service, an essential part of the Linux ecosystem, is
+worth digging in. It allows its users to keep hardware up to date with firmware
+updates. Now we are waiting for fwupd for QubesOS. Kudos to [Red Hat](https://2019.osfc.io/uploads/talk/paper/9/Introducing_the_Linux_vendor_Firmware_service.pdf) for your work.
+
+* [UEFI compliance testing](https://vimeo.com/488138727) by Heinrich Schuchardt
+
+These guys are doing great stuff for ARM and RISC-V with tools that are worthy
+of mentioning, e.g., Platform Initialization SCT and Firmware Test Suite (FWTS).
+The overall quality of UEFI-compliance test results is impressive. We liked the
+idea of UEFI Self Certification Tests, UEFI SCT compliance across various UEFI
+implementations for different platforms (QEMU, x86, RISC-V). Kudos to [U-Boot](https://github.com/u-boot/u-boot) for your work.
+
+* [coreboot Thermal Management on Intel based systems](https://vimeo.com/488143663) by Sumeet Pawnikar, Timothy Wawrzynczak
+
+Intel works on this contribution to enable efficient thermal management for
+Google Chromebooks. It is an important topic for all types of devices,
+especially those focused on energy saving, e.g., mobile platforms. The
+presenters discussed the added ACPI code and explained the code tree with its
+structure. Kudos to [Intel](https://software.intel.com/content/www/us/en/develop/articles/coreboot.html) for your piece of work.
+
+* [Bootling Linux more safely with safeboot](https://vimeo.com/488144473) by [Trammell Hudson](@qrs)
+
+Trammel, the founder, and contributor of safeboot, presented a convenient talk
+around configuring UEFI Secure Boot tools. Kudos to Trammel Hudson for his work [safeboot](https://safeboot.dev/)
+
+* [PRM: SMM Goes on a Diet](https://vimeo.com/488136678) by Michael Kubacki
+
+Great, technical talk on System Management Mode.  A few noticeable facts:
+* some hardware resources can only be accessed in SMM
+* RAS relies on SMM and SMI handlers
+* UEFI Capsule Update and Authenticated Variables
+* SMM is related to OCP and server use cases https://www.youtube.com/watch?v=mu3DRLM1dPA
```

Comment:
@KBanecka definitely some words about PRM itself is needed. @miczyg1 can you help here?

Reply:
`* PRM stands for Platform Runtime Mechanism and is designed to reduce the privileged SMM mode code`
`* Certain SMI handlers are moved to PRM modules which operating system may call to handle an event`